fabio.properties

```diff
@@ -353,6 +353,13 @@
 # proxy.header.tls.value =
 
 
+# proxy.header.requestid configures the header for the adding request id.
```

Comment:
`proxy.header.requestid configures the header for the adding a unique request id.`